File: between_limits.go
package sprint

func BetweenLimits(from, to rune) string {

	start, end := OrderRunes(from, to)

	result := ""

	for r := start + 1; r < end; r++ {
		result += string(r)
	}
	return result
}

func OrderRunes(a, b rune) (rune, rune) {
	switch {
	case a < b:
		return a, b
	case a > b:
		return b, a
	default:
		return a, b
	}
}

File: BetweenLimits.java
package sprint;

public class BetweenLimits {

    public String findRange(char from, char to){
         // Ensure that the characters are in ascending order
         if (from > to) {
            char temp = from;
            from = to;
            to = temp;
        }
         // Build the result string with all characters between from and to
         StringBuilder result = new StringBuilder();
        for (char c = (char)(from+1); c < to; c++){
            result.append(c);
        }
        return result.toString();
    }
}
